browser/components/CodeEditor.js

```diff
@@ -205,12 +205,33 @@ export default class CodeEditor extends React.Component {
         // Do nothing
       },
       [translateHotkey(hotkey.insertDate)]: function (cm) {
+        const dateNow = new Date()
+        if (self.props.dateFormatISO8601) {
+          cm.replaceSelection(dateNow.toISOString().split('T')[0])
+        } else {
+          cm.replaceSelection(dateNow.toLocaleDateString())
+        }
+      },
+      'Cmd-/': function (cm) {
```

Comment:
I think you should remove the code for `Cmd-/` and `Shift-Ctrl-/` and replace that with `[translateHotkey(hotkey.insertDateTime)]`. You code should be something like this:
```js
[translateHotkey(hotkey.insertDateTime)]: function (cm) {
  const dateNow = new Date()
  if (self.props.dateFormatISO8601) {
       cm.replaceSelection(dateNow.toISOString())
  } else {
       cm.replaceSelection(dateNow.toLocaleString())
  }
}
```